### PR TITLE
[SP-143] Moved all validators to shared project

### PR DESCRIPTION
**Pull request by innowate**

### CODE REVIEW

File: shared/test/validators/password.validator.dart [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+         cases.forEach((testCase) {
+            var password = testCase['password'];
+            var result = testCase['result'];
+
+            test('Should validate password $password', () {
+               expect(PasswordValidator.validate(control), result, reason: password);
+            });
+         });
+
+         test('Should be valid if control value is null', () {
```
Comment (Ne4istb) [inline shared/test/validators/password.validator.dart]: ❌ there is already test for null value in previous set of tests

File: shared/test/validators/user-name.validator.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+         cases.forEach((testCase) {
+            var userName = testCase['userName'];
+            var result = testCase['result'];
+
+            test('Should validate userName $userName', () {
+               expect(UserNameValidator.validate(control), result, reason: userName);
+            });
+         });
+
+         test('Should be valid if control value is null', () {
```
Comment (Ne4istb) [inline shared/test/validators/user-name.validator.dart]: ❌ there is already test for null value in previous set of tests

File: shared/test/validators/password.validator.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import 'package:test/test.dart';
+import 'package:smartpeople_shared/src/validators/password.validator.dart';
+
+class PasswordValidatorTests {
+   static run() {
+      group('PasswordValidator', () {
+         List<Map<String, dynamic>> cases = [
+            {'password': null, 'result': success},
```
Comment (Ne4istb) [inline shared/test/validators/password.validator.dart]: ✏️ replace 'success' with 'true' and 'failed' with 'false'. it more obvious.

File: shared/test/validators/user-name.validator.dart [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import 'package:test/test.dart';
+import 'package:smartpeople_shared/src/validators/user-name.validator.dart';
+
+class UserNameValidatorTests {
+   static run() {
+      group('UserNameValidator', () {
+         List<Map<String, dynamic>> cases = [
+            {'userName': null, 'result': success},
```
Comment (Ne4istb) [inline shared/test/validators/user-name.validator.dart]: ✏️ replace 'success' with 'true' and 'failed' with 'false'. it more obvious.

**Comment by Ne4istb** [comment]
@innowate Все норм, осталось слегка подчистить)

**Comment by sanya-terror** [comment]
@innowate, не забывай лейблы чистить, так сказать обновлять статус стори'

**Comment by innowate** [comment]
@sanya-terror, окей)

**Pull request merged**